Parser/Lexer/Token.cs

```diff
@@ -49,7 +49,8 @@ public enum E {
          Console.WriteLine (title);
          Console.WriteLine ("┬".PadLeft (gutter, '─').PadRight (title.Length, '─'));
          for (int i = Line - 2; i <= Line + 2; i++) {
-            if (i < 1 || i > lines.Length) continue;
+            if (i < 1) continue;
+            if (i > lines.Length) break;

```

Comment:
This is an unnecessary micro-optimization. Anyway the loop will finish in 2 more iterations, so it does not really help much by exiting early.